tests/manage/monitoring/pagerduty/test_ceph.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import managed_service_required, tier4, tier4a
+from ocs_ci.ocs import constants
+from ocs_ci.utility import pagerduty
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
```

Comment:
please place in tier4b. See https://docs.google.com/document/d/1fcUdGcmPCzQt1kMznO0u81YKF1X_XK40A5ZFUx6Vlsg/edit

Reply:
Tier markers are updated according to a new definition. PG corrupt test stays in tier4a because it introduces an error directly into Ceph and is not related to node failures.